tools/bin/check-duplicated-storages.sh

```diff
@@ -0,0 +1,41 @@
+#!/bin/bash
+
```

Comment:
Maybe it's a good idea to use
```suggestion

set -e

```
to exit immediately if a command exits with a non-zero status.